### PR TITLE
ERC20BridgeSampler: Unlock full sampling on Kovan

### PR DESCRIPTION
**Pull request by dorothy-zbornak**
## Description

Adds kovan addresses for dexes and pins the input source IDs to be constant across networks.

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/contract-addresses/addresses.json [hunk 1]
```diff
@@ -114,7 +114,7 @@
         "erc20BridgeProxy": "0xfb2dd2a1366de37f7241c83d47da58fd503e2c64",
         "uniswapBridge": "0x0000000000000000000000000000000000000000",
         "eth2DaiBridge": "0x0000000000000000000000000000000000000000",
```
Comment (fragosti) [inline packages/contract-addresses/addresses.json]: We don't need these?

File: contracts/utils/contracts/src/DeploymentConstants.sol [hunk 2]
```diff
@@ -26,22 +26,30 @@ contract DeploymentConstants {
...
+    // address constant private ETH2DAI_ADDRESS = 0x4A6bC4e803c62081ffEbCc8d227B5a87a58f1F8F   ;
     /// @dev Mainnet address of the `ERC20BridgeProxy` contract
     address constant private ERC20_BRIDGE_PROXY_ADDRESS = 0x8ED95d1746bf1E4dAb58d8ED4724f1Ef95B20Db0;
     // /// @dev Kovan address of the `ERC20BridgeProxy` contract
     // address constant private ERC20_BRIDGE_PROXY_ADDRESS = 0xFb2DD2A1366dE37f7241C83d47DA58fd503E2C64;
     ///@dev Mainnet address of the `Dai` (multi-collateral) contract
     address constant private DAI_ADDRESS = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    // ///@dev Kovan address of the `Dai` (multi-collateral) contract
+    // address constant private DAI_ADDRESS = 0x4F96Fe3b7A6Cf9725f59d353F723c1bDb64CA6Aa;
     /// @dev Mainnet address of the `Chai` contract
```
Comment (fragosti) [inline contracts/utils/contracts/src/DeploymentConstants.sol]: What is the convention here when you deploy to kovan you uncomment?

**Comment by dorothy-zbornak** [comment]
> We need to deploy the actual bridges on Kovan as well. Will this be a separate pr?

Ah, I didn't notice they weren't deployed. I'll do it here.

File: contracts/utils/contracts/src/DeploymentConstants.sol (hunk 2 above)
Comment (dorothy-zbornak) [reply contracts/utils/contracts/src/DeploymentConstants.sol]: yeah, pretty high tech

**Comment by xianny** [comment]
Would you mind updating migrations script and Ganache snapshot too? 🙏 Or create a separate task for it

**Pull request merged**